docs/getting-started/integrations/dotnet.md

```diff
@@ -20,29 +21,139 @@ You get ready-to-use code tailored to your needs!
 
 ## Setting up the project
 
-For the purpose of this guide, we will use a basic ASP.NET Core project created with `dotnet new webapp`. You can refer to the [ASP.NET Core documentation](https://learn.microsoft.com/en-us/aspnet/core/getting-started/?view=aspnetcore-7.0) to learn how to set up a project in the framework.
+For the purpose of these guides, we will use a basic ASP.NET Core project created with `dotnet new webapp`. You can refer to the [ASP.NET Core documentation](https://learn.microsoft.com/en-us/aspnet/core/getting-started/?view=aspnetcore-7.0) to learn how to set up a project in the framework.
```

Comment:
Why plural?

Reply:
My thought process was: "This section is relevant to both paths - CDN and ZIP," so I used the plural. But, as I'm thinking now, I could also treat the whole guide as one. I don't know which is better 🤔

Reply:
I suggest treating the whole guide as one, it's just a guide for laravel or dotnet and that's it. Paths are something different in the nomenclature we use, and guides are documents.